src/component/OverviewMap.js

```diff
@@ -93,13 +96,15 @@ Ext.define("GeoExt.component.OverviewMap", {
         /**
          * A configured map or a configuration object for the map constructor.
          * This is the overviewMap itself.
-         * @cfg {ol.Map} map
+         *
+         * @cfg {ol.Map/Object} map
```

Comment:
In the created doc it doesn't make a difference. Do you want me to change all these occurrences? I can do a search and replace if you want to.